stories/configMock.js

```diff
@@ -1,3 +1,5 @@
+const ONE_GiB = 2 ** 30;
```

Comment:
Trivial - This is already defined in `src/constants.js`. Can we reuse that ?

Reply:
Good catch 👍